docker-compose.dev-full.yml

```diff
@@ -143,6 +143,13 @@ services:
             - redis
             - kafka
 
+    property-defs-rs:
```

Comment:
Only complaint about this (that I also have for capture) is that it pulls from master instead of mounting the local files. If we used `build` to build the local rust image then we could _also_ dev locally using this. Should be an easy change if you have time?

Reply:
Sorry, I don't really understand what you mean by using `build`. I'm going to merge this, just to get this loop closed, but we can maybe take a couple of minutes during tomorrows sync for you to show me what you mean.